tests/test_pdf_erfexp.py

```diff
@@ -0,0 +1,89 @@
+"""Tests for ErfExp PDF."""
+
+import numpy as np
+import pytest
+import tensorflow as tf
+import zfit
+from scipy import integrate, special
+from zfit.core.testing import tester
+
+import zfit_physics as zphys
+
+alpha_true = 90.0
+beta_true = 0.08
+gamma_true = -1.0
+n_true = 0.2
+
+alpha_range = (65.0, 90.0)
+gamma_range = (-10, 10)
+beta_range = (0.01, 10)
+n_range = (0.1, 0.5)
+
+
+def _erfexp_numpy(x, alpha, beta, gamma, n):
+    return special.erfc((x - alpha) * beta) * np.exp(-gamma * (np.power(x, n) - np.power(alpha, n)))
+
+
+erfexp_numpy = np.vectorize(_erfexp_numpy, excluded=["alpha", "beta", "gamma", "n"])
+
+
+def create_erfexp(alpha, beta, gamma, n, limits):
+    obs = zfit.Space("obs1", limits=limits)
+    erfexp = zphys.pdf.ErfExp(alpha=alpha, beta=beta, gamma=gamma, n=n, obs=obs, norm=False)
```

Comment:
Oh no, it's just because I'm comparing with a vectorized numpy implementation in the tests and that s unnormalized. Nothing wrong with the normalization